pages/intro.js

```diff
@@ -0,0 +1,197 @@
+import Card, { CardHeader, CardTitle, CardContent, CardButtonContainer } from '../components/card'
+import styled from 'styled-components'
+import { useState, useEffect } from 'react'
+import { COLOR, EDIT } from '../constants'
+import Button from '../components/button'
+
+const props = {
+    nwHacks: {
```

Comment:
I don't think we will be passing in content inside a hackathon object, ie we don't need to do `nwHacks.subtitle` we should be using just `subtitle`

Reply:
im a little confused about what this means, where does th e variable `subtitle` come from? where is it passed in if not in a hackathon object?

Reply:
for example I think the data should look like
```
{
subtitle: 'lets go'
content: '-dababy'
}
```
instead of 
```
nwhacks: {
subtitle: 'lets go'
content: '-dababy'
}
```